compatibility/compatibility-tests/src/test/java/org/projectnessie/tools/compatibility/tests/ITRollingUpgradeScenario.java

```diff
@@ -98,7 +98,7 @@ void scenario() throws Exception {
   Branch commit(NessieApiV1 api, String suffix, Branch head) throws Exception {
     ThreadLocalRandom tlr = ThreadLocalRandom.current();
 
-    ContentKey key = ContentKey.of("key-" + suffix, "v" + version.toString());
+    ContentKey key = ContentKey.of("key-" + suffix + "-v" + version.toString());
```

Comment:
Just to make sure I understand: this change is just to skip creating the `"key-" + suffix` namespace, right?

Reply:
Yes.